docs/Spec.md

```diff
@@ -0,0 +1,74 @@
+### Spec
+
+---
+
+### Feature Definitions
+
+[Filter of GitHub Features](https://github.com/fsheremetyeva/capstone/labels/Feature)
+
+- Ability to create user volunteer account
+- Ability to create non-profit organization account
+- Ability to search for volunteer opportunities
+- Initial Dashboard for volunteer account
```

Comment:
What are the capabilities of this dashboard? We can see the following being feature possibilities:

- [ ] Update email
- [ ] Update Password
- [ ] Add/Modify/Remove Capabilities
- [ ] Add/Modify/Remove Availability
- [ ] Add new contact information
- [ ] Remove contact information
- [ ] Modify contact information
- [ ] Upcoming Promised Volunteer Events
- [ ] Volunteer History
- [ ] Certificates

Reply:
Thank you, I moved this to my existing issue and commented there. 

A user is able to add/edit and delete information:

- [ ] Update email
- [ ] Update Password
- [ ] Add/Modify/Remove Capabilities
- [ ] Add new contact information
- [ ] Remove contact information
- [ ] Modify contact information
- [ ] Volunteer History

Other Features:
- [ ] Pie chart summarizing volunteer activity
- [ ] Generate Certificate

Reply:
Added to Project Spec